tests/ceph_ansible/purge_cluster.py

```diff
@@ -55,6 +57,10 @@ def run(**kw):
     playbook_regex = re.search('(purge-.*?\\.yml)(.*)', playbook_command)
     playbook = playbook_regex.group(1)
     playbook_options = playbook_regex.group(2)
+
+    # Set ansible deprecation warning to false
+    installer_node.exec_command(cmd='export ANSIBLE_DEPRECATION_WARNINGS=False')
+
```

Comment:
would suggest to make if else code part more readable, like 

- prefix the command inside if else block and run necessary command inside as per the release version.
- But ansible playbook is common command which can be executed outside if else block.

Reply:
And comment at every import step in module.